config.json

```diff
@@ -76,8 +75,15 @@
           "placeholder": "testim --version",
           "required": false,
           "learnUrl": "https://help.testim.io/docs/the-command-line-cli"
+        },
+        {
+          "name": "testimInstall",
```

Comment:
I would name it `installTestim`.
That won't match the remaining parameters, but it doesn't need to, because they serve different purpose (Testim domain parameters vs action to be performed using testim).